community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -513,6 +513,14 @@ variable "enable_maintenance_reservation" {
   default     = false
 }
 
+
+variable "enable_opportunistic_maintenance" {
+  type        = bool
+  description = "Enables opportunistic maintenance to perform GCP maintenance early."
```

Comment:
Please add description of behavior. E.g. "On receiving maintenance notification, maintenance will be performed as soon as nodes becomes idle."